resources/views/forum/threads/_header.blade.php

```diff
@@ -0,0 +1,33 @@
+<h1>{{ $title }}</h1>
+<div class="thread-info">
+    <div class="thread-info-avatar">
+        @if (count($thread->replies()))
+            @php($lastReply = $thread->replies()->last())
+            <img class="img-circle" src="{{ $lastReply->author()->gratavarUrl(25) }}">
+        @else
+            <img class="img-circle" src="{{ $thread->author()->gratavarUrl(25) }}">
+        @endif
+    </div>
+
+    <div class="thread-info-content">
+        @if (count($thread->replies()))
+            @php($lastReply = $thread->replies()->last())
+            <a href="{{ route('profile', $lastReply->author()->username()) }}" class="thread-info-link">{{ $lastReply->author()->name() }}</a>
+            added the last reply {{ $lastReply->createdAt()->diffForHumans() }}
+
+        @else
+            <a href="{{ route('profile', $thread->author()->username()) }}" class="thread-info-link">{{ $thread->author()->name() }}</a>
+            posted {{ $thread->createdAt()->diffForHumans() }}
+        @endif
```

Comment:
You can remove the if statement and just keep the content of the `else` part. It doesn't really matters who posted the last reply.